packages/react-components/src/cascader/cascader-footer.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from '../button';
+import { Flex } from '../flex';
+import { CascaderOption } from './cascader';
+import { SelectionPath } from './cascader-provider';
+import { useCascaderValue } from './use-value';
+
+export const CascaderFooter = ({
+  handleChange
+}: {
+  handleChange: (value: CascaderOption, selectionPath: SelectionPath[]) => void;
+}) => {
+  const { selectionPath, selectedNode, closePopover } = useCascaderValue();
+
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="space-between"
+      css={{
+        padding: '$4 $6',
+        width: '100%',
+        borderTop: '1px solid $colors$neutral100'
+      }}
+    >
+      <Button
+        size="md"
+        variant="ghost"
+        color="default"
+        onClick={() => {
+          closePopover();
+        }}
+      >
+        Cancel
```

Comment:
here as well, for CTAs labels will need i18n support. Also in the demo video, it felt like the vertical spacing of the footer is off. Space above the actions to the border above is less compared to the space from actions to the end of footer

![image](https://github.com/user-attachments/assets/52e85f4b-e96d-45c9-9a13-7524e21e602a)